### PR TITLE
ENH: Add --fs-subjects-dir flag

### PR DESCRIPTION
**Pull request by effigies**
Working toward #27.

### CODE REVIEW

File: smriprep/workflows/base.py [hunk 1]
```diff
@@ -129,6 +133,8 @@ def init_smriprep_wf(
                 spaces=[s for s in output_spaces.keys() if s.startswith('fsaverage')] + [
                     'fsnative'] * ('fsnative' in output_spaces)),
             name='fsdir_run_%s' % run_uuid.replace('-', '_'), run_without_submitting=True)
+        if fs_subjects_dir is not None:
+            fsdir.inputs.subjects_dir = str(fs_subjects_dir.absolute())
```
Comment (oesteban) [inline smriprep/workflows/base.py]: Is `absolute()` better than `resolve()` for some reason? (this is a question, not a change suggestion here).

**Comment by oesteban** [comment]
Is poldracklab/niworkflows#398 released? Do you need to update some pin version?

File: smriprep/workflows/base.py (hunk 1 above)
Comment (effigies) [reply smriprep/workflows/base.py]: It doesn't attempt to resolve symlinks, just turns relative paths into absolute.

**Comment by effigies** [comment]
> Is [poldracklab/niworkflows#398](https://github.com/poldracklab/niworkflows/pull/398) released? Do you need to update some pin version?

Good catch. Thanks.

File: smriprep/workflows/base.py (hunk 1 above)
Comment (oesteban) [reply smriprep/workflows/base.py]: Thanks for the clarification - pretty useful.

**Comment by oesteban** [comment]
Do you want to test this new flag on circle? (i.e., update the ds005 build to exercise it)

**Comment by oesteban** [comment]
I think a new section under Usage or Q in the FAQ regarding the reuse of FreeSurfer derivatives could be a great improvement. Also, some updates to https://fmriprep.readthedocs.io/en/latest/outputs.html#freesurfer-derivatives could be good to have.

Probably just linking https://github.com/poldracklab/fmriprep/issues/1573 would satisfy my request. Merging.

**Comment by effigies** [comment]
Oh sure. When we add this to fMRIPrep, we should definitely update those docs. Just not much to do here in sMRIPrep that I can see.

**Pull request merged**